clang/tools/driver/CacheLauncherMode.cpp

```diff
@@ -189,6 +189,10 @@ clang::handleClangCacheInvocation(SmallVectorImpl<const char *> &Args,
       Args.append({"-Wno-builtin-macro-redefined", "-D__DATE__=\"redacted\"",
                    "-D__TIMESTAMP__=\"redacted\"", "-D__TIME__=\"redacted\""});
     }
+    if (llvm::sys::Process::GetEnv(
+            "CLANG_CACHE_CHECK_REPRODUCIBLE_CACHING_ISSUES")) {
+      Args.append({"-Werror=reproducible-caching"});
```

Comment:
Why is it behind environment variable check instead of default on for cached builds?

Reply:
It broke WebKit build which uses `-Werror` and `__TIMESTAMP__`.